src/import/config.rs

```diff
@@ -65,6 +67,9 @@ impl Config {
         if self.max_open_engines == 0 {
             return Err("import.max_open_engines can not be 0".into());
         }
+        if self.min_available_ratio < 0.02 {
```

Comment:
I don't think we need to impose a hard limit of 2%. Just checking `< 0` is sufficient.